webpack/loaders.js

```diff
@@ -1,6 +1,4 @@
-'use strict';
-
-const ExtractTextPlugin = require('extract-text-webpack-plugin');
+var ExtractTextPlugin = require('extract-text-webpack-plugin');
```

Comment:
vars usage violates our style-guide,
Please use const where possible, and let when mutation is neseccary